lib/util/api-json.js

```diff
@@ -63,7 +63,11 @@ function api(token, endpoint, method, requestBody, callback) {
                 if (responseBody.trim().length === 0) {
                     responseBody = null;
                 } else {
-                    responseBody = JSON.parse(responseBody);
+                    try {
+                        responseBody = JSON.parse(responseBody);
+                    } catch (e) {
+                        return callback(new APIError(e, responseBody, responseCode));
```

Comment:
If we don't `return` here, the rest of the code will execute as well (and so another `callback()` would run).  No one is expecting an actual return value from this function, they're waiting on a callback, so instead of having `callback()` and `return` on two lines I often combine them into one.

Yeah it looks like `APIError` will set its `.httpMessage` and `.message` to the Error object.  Guess I could change it to `e.message` instead.